README.md

```diff
@@ -1,21 +1,48 @@
-# Devextreme UI Template Gallery
+# DevExtreme UI Template Gallery
 
-This repository includes responsive UI Templates for the most pupular UI/UX patterns in web LOB applications. Currently, this project is in the 'preview' stage and contains the following CRM and Project Management UI templates:
+This repository includes responsive UI Templates for the most popular UI/UX patterns in web LOB applications. These templates are available for Angular, React, and Vue.
 
-* [CRM. Contact List](https://devexpress.github.io/devextreme-ui-template-gallery/material.blue.light.compact/angular/default/#/crm-contact-list)
-* [CRM. Contact Details](https://devexpress.github.io/devextreme-ui-template-gallery/material.blue.light.compact/angular/default/#/crm-contact-details)
-* [Planning. Task List](https://devexpress.github.io/devextreme-ui-template-gallery/material.blue.light.compact/angular/default/#/planning-task-list)
-* [Planning. Task Details](https://devexpress.github.io/devextreme-ui-template-gallery/material.blue.light.compact/angular/default/#/planning-task-details)
+![DevExtreme UI Template Gallery](/images/ui-template-gallery.png)
+
+Take a look at the gallery **online** here:
+
+- [Angular](https://devexpress.github.io/devextreme-ui-template-gallery/material.blue.light.compact/angular/default/#/crm-contact-list)
+- [React](https://devexpress.github.io/devextreme-ui-template-gallery/material.blue.light.compact/react/default/#/crm-contact-list)
+- [Vue](https://devexpress.github.io/devextreme-ui-template-gallery/material.blue.light.compact/vue/default/#/crm-contact-list)
+
+Use the left menu to navigate through the CRM, Planning and Analytics views.
 
 The UI Templates have responsive layouts with DevExtreme UI components. Note that the components use sample data entities without any business logic. You can use these UI Templates in your project and adapt them to your specific business requirements.
 
-> The repository is in preview mode. Only Angular version is available for use. Vue and React projects are still under development.
+## Get Started
+
+To run these UI templates on your machine, clone this repository and follow the instructions below.
+
+1. Choose a framework and go to a corresponding folder:
```

Comment:
@vladaskorohodova  Sorry, my bad.
I've just looked carefully and realized that I wrote it incorrectly
The actual step sequence is
1. Install modules with `npm i` command
2. Go to a desired framework folder
3. Call the run script
4.  Open the application on the local server. The link will be in the console output.